### PR TITLE
[ko] add korean translations

### PR DESCRIPTION
**Pull request by derornos**
Initial work on Korean translations.

Please let me know if you find any issues.

thank you.

### CODE REVIEW

File: src/assets/locales/ko.json [hunk 1]
```diff
@@ -0,0 +1,1143 @@
...
+      "out": "축소",
+      "reset": "확대/축소 초기화"
+    }
+  },
+  "spoiler": {
+    "add": "스포일러 항목 추가",
+    "all": "모든 스포일러 항목 공개하기",
+    "placeholder": "???"
+  },
+  "state": {
```
Comment (derornos) [inline src/assets/locales/ko.json:894]: Where can I find this "state" entry and its sub-entries?

Comment (Lurkars) [reply src/assets/locales/ko.json:894]: Mean this https://github.com/Lurkars/gloomhavensecretariat/blob/main/src/assets/locales/en.json#L893?

**Comment by Lurkars** [comment]
Cool, I cannot verify of course. But looking good! There are also some label in edition data, see https://github.com/Lurkars/gloomhavensecretariat/blob/main/data/gh/label.json for example

File: src/assets/locales/ko.json (hunk 1 above)
Comment (derornos) [reply src/assets/locales/ko.json:894]: Yes, that's right.

I wonder where the [corresponding item](https://github.com/Lurkars/gloomhavensecretariat/blob/main/src/assets/locales/en.json#L893) is being displayed on the website.

[https://gloomhaven-secretary.de/](https://gloomhaven-secretary.de/)

**Comment by derornos** [comment]
Yes. Currently working on it.
thank you for telling me.

File: src/assets/locales/ko.json (hunk 1 above)
Comment (Lurkars) [reply src/assets/locales/ko.json:894]: Ah okay. That are the texts for undo/redo! So those appear in main menu after taken corresponding actions.

Comment (derornos) [reply src/assets/locales/ko.json:894]: i got it.

thanks : )

**Comment by Lurkars** [comment]
Just ping me here, when you think it's ready to merge

**Comment by derornos** [comment]
Please merge.
thanks : )

**Pull request merged**